docs/querying/sql.md

```diff
@@ -49,6 +49,7 @@ Druid SQL supports SELECT queries with the following structure:
 [ WITH tableName [ ( column1, column2, ... ) ] AS ( query ) ]
 SELECT [ ALL | DISTINCT ] { * | exprs }
 FROM { <table> | (<subquery>) | <o1> [ INNER | LEFT ] JOIN <o2> ON condition }
+[ FILTER ( WHERE condition ) ]
```

Comment:
this is an optional part of SELECT and also HAVING, so I guess more accurate would be removing this newly added line and changing those lines to:

```
SELECT [ ALL | DISTINCT ] { * | exprs } [ FILTER ( WHERE expr ) ]
```
and
```
[ HAVING expr [ FILTER ( WHERE expr ) ] ]
```
I think?

Reply:
actually wondering if i messed the select one up and it should be 

```
SELECT [ ALL | DISTINCT ] { * | exprs [ FILTER ( WHERE expr ) ] } 
```

but that’s not really quite right either, sort of would need to expand `exprs` into 

```
expr1 [ FILTER ( WHERE expr ) ], expr2 ...
```

maybe its fine as it is, though it sort of reads like you could write `SELECT * FILTER (WHERE ...) FROM ... ``` which you can't